src/lib/fs/read.js

```diff
@@ -0,0 +1,31 @@
+import chalk from 'chalk';
+import { createReadStream, statSync } from 'node:fs';
+import path from 'node:path';
+
+export default function readFile(fileName) {
+  try {
+    const resolvedPath = path.resolve(fileName);
+    const stats = statSync(resolvedPath);
+
+    if (stats.isDirectory()) {
+      console.error(chalk.red('Error: Can not read a directory'));
+      return;
+    }
+
+    const readStream = createReadStream(resolvedPath, { encoding: 'utf-8' });
+
+    readStream.on('data', (chunk) => {
+      console.log(chalk.yellowBright('Part of the data:\n'), chunk);
+    });
+
+    readStream.on('end', () => {
+      console.log(chalk.green('File reading completed.'));
+    });
+
+    readStream.on('error', () => {
+      console.error(chalk.red('Error reading file (in stream):'), err.message);
+    });
+  } catch (err) {
+    console.error(chalk.red('Error creating stream:'), err.message);
```

Comment:
why here we don't throw an error like in other files?

Reply:
I just log here an error, and if we will have an error we will catch it on upper levels, where we call executedCommands method